_posts/2024-04-02-shared-conda-tutorial.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: The ideal setup for a multi-user conda installation
+tags: conda bash systems-administration
```

Comment:
Maybe better as a list here. What do you think?

Reply:
Both are acceptable from the frontmatter perspective, they're treated equivalently by `jekyll-tagging`.